diff --git a/cmake/modules/contrib/VitisAI.cmake b/cmake/modules/contrib/VitisAI.cmake
index 083bd6d..c39419b 100644
--- a/cmake/modules/contrib/VitisAI.cmake
+++ b/cmake/modules/contrib/VitisAI.cmake
@@ -17,24 +17,24 @@
 
 if(USE_VITIS_AI)
   set(PYXIR_SHARED_LIB libpyxir.so)
-  find_package(PythonInterp 3.6 REQUIRED)
-  if(NOT PYTHON)
-    find_program(PYTHON NAMES python3 python3.6)
-  endif()
-  execute_process(COMMAND "${PYTHON_EXECUTABLE}" "-c"
-    "import pyxir as px; print(px.get_include_dir()); print(px.get_lib_dir());"
-    RESULT_VARIABLE __result
-    OUTPUT_VARIABLE __output
-    OUTPUT_STRIP_TRAILING_WHITESPACE)
-
-  if(__result MATCHES 0)
-    string(REGEX REPLACE ";" "\\\\;" __values ${__output})
-    string(REGEX REPLACE "\r?\n" ";"    __values ${__values})
-    list(GET __values 0 PYXIR_INCLUDE_DIR)
-    list(GET __values 1 PYXIR_LIB_DIR)
-  else()
-    message(FATAL_ERROR "Can't build TVM with Vitis-AI because PyXIR can't be found")
-  endif()
+  #find_package(PythonInterp 3.6 REQUIRED)
+  #if(NOT PYTHON)
+  #  find_program(PYTHON NAMES python3 python3.6)
+  #endif()
+  #execute_process(COMMAND "${PYTHON_EXECUTABLE}" "-c"
+  #  "import pyxir as px; print(px.get_include_dir()); print(px.get_lib_dir());"
+  #  RESULT_VARIABLE __result
+  #  OUTPUT_VARIABLE __output
+  #  OUTPUT_STRIP_TRAILING_WHITESPACE)
+  #
+  #if(__result MATCHES 0)
+  #  string(REGEX REPLACE ";" "\\\\;" __values ${__output})
+  #  string(REGEX REPLACE "\r?\n" ";"    __values ${__values})
+  #  list(GET __values 0 PYXIR_INCLUDE_DIR)
+  #  list(GET __values 1 PYXIR_LIB_DIR)
+  #else()
+  #  message(FATAL_ERROR "Can't build TVM with Vitis-AI because PyXIR can't be found")
+  #endif()
   message(STATUS "Build with contrib.vitisai")
   include_directories(${PYXIR_INCLUDE_DIR})  
   file(GLOB VAI_CONTRIB_SRC src/runtime/contrib/vitis_ai/*.cc)
